src/Chapter1.hs

```diff
@@ -503,7 +504,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = min (abs x) (abs y)
```

Comment:
This implementation is almost correct. Unfortunately, for a negative number, it returns its absolute value instead of the negative number itself. So, `closestToZero (-7) 3` will return `7` instead of `-7`.

Reply:
Damn it, I kind of overread that subtlety :)